### DISCUSSION TITLE
HW acceleration support status for arm64v8 and AMD/ATI ?

### DISCUSSION THREAD

**Discussion by btoplak** [datarhei/restreamer#860] [General]
Hi there!

I just stumbled on Restreamer a few days ago and I was testing it since... I want to congratulate all the authors, you're doing a very good job in making video streaming a breeze for a novice users, but also a very useful tool for us more advanced users with some experience!

I'm investigating various devices where I could run the Restreamer on. On your Docs page I found that you offer Docker images for ARM, but only arm32v7 is marked as "HW accelerated". I have several arm64v8 devices laying around, could those also run _datarhei/restreamer:rpi-latest_ image to enjoy the HW acceleration? If not, do you plan to prepare a HW accel image for this platform too?

Another question is regarding HW acceleration on AMD CPUs with integrated ATI GPUs. Are there any plans (or maybe test Docker images) for using the ffmpeg's support for AMD/ATI GPUs for HW acceleration?

Thanks!

**Comment by ioppermann** [comment]
The arm32v7 build is targetted specifically to a Raspberry Pi. We are looking into providing Vulkan support for HW acceleration. Currently we only provide CUDA images for nvidia GPUs.

**Comment by btoplak** [comment]
Ok, thank you very much for your feedback.

If the core is based on ffmpeg I think I could take look at it and try to make adjustments to use the HW encoding inbuilt in latest ffmpeg lib.
